config/packages/dev/mailer.yaml

```diff
@@ -0,0 +1,5 @@
+framework:
+    mailer:
+        dsn: 'null://null'
```

Comment:
why not import from env like the main one? This would be easier to manage with a local file.